.ado/pipelines/config/loadtest-baseline-normal.json

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "passFailMetrics": {
+            "ece127bc-6271-4a70-93f6-758c56a3f11c": {
```

Comment:
and I guess they dont have to be unique?!

Reply:
Not clear. Needs further investigation. I'd prefer to auto-generate them instead of defining them in our baseline json at some point.

Reply:
maybe just make sure they are at least unique in our two baseline files. no need to generate them on the fly in the pipeline, I guess

Reply:
made our sample guids unique